src/main/core/main.rs

```diff
@@ -178,6 +219,10 @@ pub fn run_shadow(build_info: &ShadowBuildInfo, args: Vec<&OsStr>) -> anyhow::Re
     unsafe { c::main_logBuildInfo(build_info) };
     log_environment(args.clone());
 
+    if let Err(e) = verify_supported_system() {
+        log::error!("Couldn't verify supported system: {e:?}")
+    }
```

Comment:
nit: maybe `warn` instead of `error` since it's not something that will cause shadow to exit (not a hard error)?

Reply:
> fwiw at some point I think we decided to use error for things that strongly indicate that the simulation results may be invalid.

I though it was the opposite, that we would never use the error log level since we could return an `anyhow::Result` in that case and bubble it up to the top. In the C code I think shadow used to always use the error log level when it was about to exit (for example if an assertion failed).

But yeah I agree we should have this documented. I don't think it was documented since we never really reached a consensus. There are some notes from that discussion in the riseup pad from 2022-03-24.